content/doc/developer/publishing/requesting-hosting.adoc

```diff
@@ -0,0 +1,59 @@
+---
+title: Guide to Plugin Hosting
+layout: developerguide
+---
+
+This guide will cover the entire plugin hosting process.
+
+Once you've completed all steps described on this page you will have accomplished the following:
+
+* Your plugin source code will be hosted in a repository in the +jenkinsci+ GitHub organization.
+  You will have admin access to this repository.
+* You will be allowed to release the plugin to the Jenkins project Maven repository, which serves as a source for the Jenkins project operated update sites.
+
+== Preparation
+
+Complete the following steps before requesting plugin hosting with the Jenkins project:
+
+. Review the link:../preparation[preparation steps] and make sure they're satisfied.
+. Make sure your plugin follows the plugin naming convention outlined in the link:../style-guides[style guides].
+. Have a public repository containing the plugin source code on GitHub.
+
+
+== Open HOSTING request
+
+Log in to link:https://issues.jenkins-ci.org/[JIRA] and create a new issue link:https://issues.jenkins-ci.org/browse/HOSTING[in the HOSTING] project.
+Make sure to fill out all fields as described.
+
+A Jenkins project member will review your request within a few days.
+If any changes are requested, please implement them.
+
+Once all requirements have been satisfied, your repository will be forked into the +jenkinsci+ organization and you will be invited to join it, and a JIRA component in the JENKINS project will be created for you.
```

Comment:
Add "you will generally be asked to _ideally_ delete the original repo so that the Jenkinsci one becomes the canonical repository", and link to source-code-hosting.adoc from here ?